<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlaveInfoFrame</class>
 <widget class="QFrame" name="SlaveInfoFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_Image">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_ValueP">
       <property name="text">
        <string>ValueP(W)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_ValueP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ValueQ">
       <property name="text">
        <string>ValueQ(Var):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ValueQ"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_ValueU">
       <property name="text">
        <string>ValueU(V):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_ValueU"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_ValueI">
       <property name="text">
        <string>ValueI(A):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_ValueI"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
